README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# About Me
+Hello, my name is Tianyi Qian.  
+
+### Educational Background  
+
+---
+
+I'm a grad student in the MAPSS program at the University of Chicago and I concentrate in **Quantitative Methods and Social Anlaysis**. I studied **Sociology** and **Media Studies** as an undergrad. I'm hoping to learn the following from this class:
```

Comment:
Improvement: I suggest that you can use bold and italic to make your words more organized and stress the points on your page